asynctnt/connection.py

```diff
@@ -330,6 +425,8 @@ async def full_connect():
                     logger.debug("connect is cancelled")
                     self._reconnect_task = None
                     raise
+                except ssl.SSLError as e:
+                    raise SSLError(e)
```

Comment:
Actually I just noticed this thing. This is a debatable solution. Why did you decide to specifically catch SSLError and stop the reconnection process? I see a potential problem here. For example, let's say everything works fine, then Tarantool server is reconfigured but incorrectly which leads to ssl errors while connecting and connection tries to reestablish in the background. And this except block raises exception without a possibility to wait for the reconnect process giving a chance to reconfigure Tarantool once more.

Reply:
Sorry, the idea was just to wrap all SSL-related errors to our `SSLError` class. I reworked the code so ssl errors are processed as usual exception.